### PR TITLE
Tag latest create-cloudflare version

### PR DESCRIPTION
**Pull request by jspspike**
Not sure the best way to about testing this, I tested with act locally and the syntax seems to work properly. Since this happens after the release if it fails it shouldn't affect the release process

**Author has included the following, where applicable:**

- [ ] Tests
- [ ] Changeset ([Changeset guidelines](https://github.com/cloudflare/workers-sdk/blob/main/CONTRIBUTING.md#changesets))

**Reviewer is to perform the following, as applicable:**

- Checked for inclusion of relevant tests
- Checked for inclusion of a relevant changeset
- Checked for creation of associated docs updates
- Manually pulled down the changes and spot-tested

**Note for PR author:**

We want to celebrate and highlight awesome PR review! If you think this PR received a particularly high-caliber review, please assign it the label `highlight pr review` so future reviewers can take inspiration and learn from it.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -57,3 +57,27 @@ jobs:
...
+            if (package.name === "create-cloudflare") {
+              const versionStr = package.version;
+              const version = versionStr.split(".");
+              const tag = `v${version[0]}-latest`;
+              const packageVersion = `create-cloudflare@${versionStr}`;
+              return {tag: tag, version: packageVersion};
+            }
+          }
+
+          return null;
```
Comment (JacobMGEvans) [inline .github/workflows/release.yml:79]: You can move this into a `.js` file at the root of `.github` and call it here similarly to this 
https://github.com/cloudflare/workers-sdk/blob/37f8731bcdbc3088073cd6fb0909bdc47aac98b3/.github/workflows/release.yml#L49

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -57,3 +57,27 @@ jobs:
...
           # Include this here because this step will rebuild Wrangler and needs to have this available
           SPARROW_SOURCE_KEY: "50598e014ed44c739ec8074fdc16057c"
+
+      - name: Get create-cloudflare tag
+        id: create-cloudflare-latest-tag
+        uses: actions/github-script@v6
+        with:
+        script: |
+          const publishedPackages = ${{ steps.changesets.outputs.publishedPackages }};
+          console.log(publishedPackages);
```
Comment (RamIdeas) [inline .github/workflows/release.yml:67]: do we want to keep this log?

**Review by RamIdeas — APPROVED** [review]

**Comment by RamIdeas** [comment]
Was gonna suggest you add the changes to wrangler to run C3 with this tag but I guess we should merge this as-is first and test it works and make that change in a follow-up PR

**Comment by lrapoport-cf** [comment]
hey @jspspike , did you still want to get this PR landed, or should it be closed?

**Comment by jspspike** [comment]
This is no longer relevant based on discussion [here](https://github.com/cloudflare/workers-sdk/pull/3903#issuecomment-1756209065)

**Pull request closed without merging**